### PR TITLE
Permissions in Android Lollipop

### PR DESCRIPTION
**Pull request by bezmian**

### CODE REVIEW

File: app/src/main/java/agency/tango/materialintro/CustomSlide.java [hunk 1]
```diff
@@ -31,4 +31,14 @@ public boolean canMoveFurther() {
     public String cantMoveFurtherErrorMessage() {
         return getString(R.string.error_message);
     }
+
+    @Override
+    public String[] possiblePermissions() {
+        return null;
```
Comment (radzio) [inline app/src/main/java/agency/tango/materialintro/CustomSlide.java]: do we need this after merging #75?

File: material-intro-screen/src/main/java/agency/tango/materialintroscreen/fragments/SlideFragmentBase.java [hunk 2]
```diff
@@ -106,6 +115,10 @@ private boolean hasPermissionsToGrant(String[] permissions) {
         return list.toArray(new String[list.size()]);
     }
 
+    private boolean isAndroidVersionSupportingPermissions() {
+        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.M;
+    }
+
     public static boolean isNotNullOrEmpty(String string) {
```
Comment (radzio) [inline material-intro-screen/src/main/java/agency/tango/materialintroscreen/fragments/SlideFragmentBase.java]: maybe use TextUtils.isEmpty from Android SDK?

**Review by radzio — CHANGES_REQUESTED** [review]

**Comment by bezmian** [comment]
rtl_support
Fixed gif 
image
Unfixed screenshot

**Comment by radzio** [comment]
Maybe we should hide permission buttons on SDK < M?

**Comment by bezmian** [comment]
@radzio Thats how it works. Button will be visible only when there is some permissions requested and Android API is >= M

**Pull request merged**